### DISCUSSION TITLE
Fitur Hapus Soap

### DISCUSSION THREAD

**Discussion by fathur23-n** [basoro/Khanza-Lite#151] [Ideas]
Izin request dokter/teman2 yang lain. Bagaimana ya script agar fitur hapus soap hanya bisa dihapus oleh dokter/yang bersangkutan. Untuk saat ini tombol bisa dihapus oleh all user, barangkali ada temen2 yang sudah modif. Terimakasih :)

**Comment by basoro** [comment]
Silahkan cek disini

https://github.com/basoro/mlite/commit/ac09618793755e817b64c18f5382702b67e7d2c8

dan disini

https://github.com/basoro/mlite/commit/a08c7c3a9617665a0e32772587ef511ce51dd29e

Atau silahkan update manual pada

https://github.com/basoro/mlite/tree/a08c7c3a9617665a0e32772587ef511ce51dd29e

Salam
